### PR TITLE
Validate excel, parse excel document in correct format

### PR DESCRIPTION
**Pull request by dtaylor7**

### CODE REVIEW

File: test/unit/services/xlsx.test.js [hunk 1]
```diff
@@ -2,23 +2,85 @@ const { expect, sinon } = require('test/unit/util/chai');
 const xlsx = require('xlsx');
```
Comment (gds-gharper) [inline test/unit/services/xlsx.test.js:2]: This is a good start, but I'd like some tests on an actual xls file read from the filesystem as well if possible.  Ideally a few, some from windows and mac.

Comment (dtaylor7) [reply test/unit/services/xlsx.test.js:2]: Would I be able to do that retrospectively so ahdab can start integrating with this?

Comment (gds-gharper) [reply test/unit/services/xlsx.test.js:2]: If you create a ticket for it then I'll approve.

Comment (dtaylor7) [reply test/unit/services/xlsx.test.js:2]: https://technologyprogramme.atlassian.net/browse/DDB-244

**Pull request merged**